templates/universal/universal.css

```diff
@@ -0,0 +1,122 @@
+.main-content-container {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.top-ad-section {
+  width: 100%; 
+  background-color: #e0e0e0;
+  padding: 20px;
+  margin-bottom: 20px;
+  border-top: 2px solid #000000;
+  border-bottom: 2px solid #ed1c24;
+}
+
+.content-and-ads-container {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.content-and-ads-container .content-section {
+  flex: 0.75; 
+  padding: 20px;
+}
+
+.content-and-ads-container .right-ad-section {
+  flex: 0.25;
+  padding: 20px;
+}
+
+.content-and-ads-container .content-section,
+.content-and-ads-container .right-ad-section {
+  width: 100%; 
+  padding: 15px; 
+}
+
+.bottom-ad-section {
+  position: fixed;
+  bottom: 0px;
+  max-height: 134px;
+  width: 992px;
+  z-index: 500;
+  left: 50%;
+  margin-left: -496px;
+  background-color: rgba(230, 230, 230, 0.7);
+}
+
+@media (min-width: 576px) {
+  .content-and-ads-container .content-section,
+  .content-and-ads-container .right-ad-section {
+    width: 100%;  /* col-sm-12 */
+  }
+}
+
+@media (min-width: 768px) {
+  .content-and-ads-container {
+    flex-direction: row;
+  }
+
+  .content-and-ads-container .content-section {
+    width: 75%;  /* col-md-9 */
+  }
+  .content-and-ads-container .right-ad-section {
+    width: 25%;  /* col-md-3 */
+  }
+}
+
+@media (max-width: 768px) {
+  .bottom-ad-section {
+    position: fixed;
+    left: 0;
+    bottom: 0;
+    min-height: 50px;
+    width: 380px;
+    min-width: 380px;
+    z-index: 500;
+    left: 50%;
+    margin-left: -190px;
+    background-color: aqua;
+  }
+}
+
+@media (min-width: 769px) {
```

Comment:
There are two media queries above for `768px` - can we combine all 3 of these into a single query? Also, does it need to be 769 and 768, or can we just use one?

Reply:
Hey, yep the min-widths can be combined, however, I have one max-width query for the bottom nav